EIPS/eip-4671.md

```diff
@@ -0,0 +1,245 @@
+---
+eip: 4671
+title: Badge Standard
+description: A standard interface for badges.
+author: Omar Aflak (@omaraflak), Pol-Malo Le Bris, Marvin Martin (@MarvinMartin24)
+discussions-to: https://ethereum-magicians.org/t/eip-4671-non-tradable-token/7976
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-13
+requires: 165
+---
+
+# Abstract
```

Comment:
```suggestion
## Abstract
```
The title hierarchy appears to be off by one. Please fix this throughout the EIP.

Reply:
Thank you for taking the time to look!